src/FastRoute/src/DBWrapper.cpp

```diff
@@ -1366,12 +1366,10 @@ void DBWrapper::fixAntennas(std::string antennaCellName,
 
 void DBWrapper::legalizePlacedCells()
 {
-  _opendp = new opendp::Opendp();
-  _opendp->init(_db);
-  _opendp->setPaddingGlobal(2, 2);
-  _opendp->detailedPlacement(0);
-  _opendp->checkPlacement(false);
-  delete _opendp;
+  opendp::Opendp* opendp = _openroad->getOpendp();
```

Comment:
this doesn't look like it should be part of dbwrapper.
in fact, I am not convinced that dbwrapper should even exist.
there is no logical separation of the db from fastroutekernel